lua/damagelogs/server/rdm_manager.lua

```diff
@@ -679,11 +679,45 @@ hook_Add("PlayerInitialSpawn", "PlayerInitialSpawn_RDM_Manager", function(ply)
     end)
 end)
 
+local function CheckRevive(ply)
+    if CR_VERSION and ply:IsRespawning() then
```

Comment:
`plymeta:IsRespawning` is not available yet (it will be in the next beta). To be safe, this should probably do
```suggestion
    if CR_VERSION and ply.IsRespawning and ply:IsRespawning() then
```

Reply:
This is no longer an issue as it has been pushed to both beta and release